src/main/java/gov/nasa/pds/objectAccess/table/TableBinaryAdapter.java

```diff
@@ -141,12 +141,19 @@ private void expandGroupField(GroupFieldBinary group, int outerOffset) {
 		int actualGroupLength = getGroupExtent(group);
 
 		if (groupLength < actualGroupLength) {
-			System.err.println("WARNING: GroupFieldBinary attribute group_length is smaller than size of contained fields: "
+			System.err.println("\nWARNING: GroupFieldBinary attribute group_length is smaller than size of contained fields: "
```

Comment:
@hhlee445 it actually looks like PDS4 JParser was handling this incorrectly before. the group length from the label and the group length from the product itself must always be `==`:

```
if (groupLength < actualGroupLength) {
    throw ParseException("Group Length mismatch: 
					+ (groupLength * group.getRepetitions().intValueExact())
					+ "!="
					+ (actualGroupLength * group.getRepetitions().intValueExact()));
}
```

This will then need to be caught somewhere in validate.

Reply:
Additionally, in the PR, since it is in a separate repo, can we be sure to cross-ref to the other repo so we know how it is being closed out?